### PR TITLE
Code of Conduct

### PR DESCRIPTION
**Pull request by joebell93**
## Description
Utilise GitHub's Code of Conduct template generator to create a "Contributor Covenant Code of Conduct"

## Related Issue
https://github.com/sky-uk/toolkit/issues/242

## Motivation and Context
Toolkit 2.0 progression

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (skitson) [inline CODE_OF_CONDUCT.md]: 'Making' - would 'make' be better? Also, is 'body size' a thing?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
```
Comment (skitson) [inline CODE_OF_CONDUCT.md]: Three separate points?

- Trolling
- Insulting/derogatory comments
- Personal or political attacks

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
```
Comment (skitson) [inline CODE_OF_CONDUCT.md]: Is sexualized the American spelling? Maybe sexualised?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (claredevine) [inline CODE_OF_CONDUCT.md]: IMHO I don't believe we should say 'body size', this gets covered in personal appearance, and its not really a term that people use I don't think.

Comment (gregorypratt) [inline CODE_OF_CONDUCT.md]: There's actually no need to say anything after "everyone"

Comment (joebell93) [inline CODE_OF_CONDUCT.md]: Some good suggestions here, this was just an auto-generated template from GitHub! Happy to amend

Comment (joebell93) [inline CODE_OF_CONDUCT.md]: imo "everyone" is too broad, there's no harm in covering a vast range of areas

**Pull request merged**